bat/tests/config-conversion/configs/1.1_builder.conf

```diff
@@ -22,3 +22,4 @@
   LOCAL_RPM_DIR = "/home/clr/mix/local-rpms"
   DOCKER_IMAGE_PATH = "clearlinux/mixer"
   OS_RELEASE_PATH = ""
+  LOG_FILE = ""
```

Comment:
I'm changing my mind on the name for this. Lets call it "LOG" just to keep it in sync with the flag name --log.